packages/app/src/systems/Bridge/components/BridgeTxItem/BridgeTxItem.tsx

```diff
@@ -28,48 +28,60 @@ export const BridgeTxItem = ({
   status,
 }: BridgeTxItemProps) => {
   return (
-    <CardList.Item onClick={onClick} css={styles.listItem}>
-      <Box.Flex align="center" css={styles.wrapper}>
+    <CardList.Item onClick={onClick} css={styles.listItem} variant="outlined">
+      <Box.Flex css={styles.wrapper}>
```

Comment:
in mobile should look to something like this:
<img width="386" alt="image" src="https://github.com/FuelLabs/fuels-portal/assets/8636507/6b0ac24e-ff7d-4276-8f99-f94079ab847c">

Reply:
and in desktop:
<img width="1070" alt="image" src="https://github.com/FuelLabs/fuels-portal/assets/8636507/083d08fe-2df2-42ce-99d6-88ebcb62d90f">